frontend/app/forms/motion-to-waive-fees/motion-to-waive-fees.web.component.js

```diff
@@ -3,21 +3,233 @@ import { Scoped } from "kremling";
 import TextInput from "../inputs/text-input.component.js";
 import FormThatPrints from "../inputs/form-that-prints.component.js";
 import Section from "../inputs/section.component.js";
+import Select from "../inputs/select.component.js";
+import Radio from "../inputs/radio.component.js";
+import GroupSelect from "../inputs/group-select.component.js";
+import CheckBox from "../inputs/checkbox.component.js";
+import TextArea from "../inputs/text-area.component.js";
+import {
+  DistrictCourtList,
+  JusticeCourtList,
+  courtTypeOptions
+} from "../form-common-options/form-common-options.js";
 
-export default class MotionToWaiveFees_Web extends React.Component {
-  render() {
-    return (
-      <Scoped css={css}>
-        <FormThatPrints>
-          <Section name="1. Personal information">
-            <TextInput dataKey="person.firstName" label="First name" />
-            <TextInput dataKey="person.middleName" label="Middle name" />
-            <TextInput dataKey="person.lastName" label="Last name" />
-          </Section>
-        </FormThatPrints>
-      </Scoped>
-    );
+const waiveFeeRelationship = [
+  { value: "plantiffPetitioner", label: "Plantiff / Petitioner" },
+  {
+    value: "plantiffPetitionerAttorney",
+    label: "Plantiff / Petitioner's Attorney"
+  },
+  { value: "defendantRespondent", label: "Defendant / Respondent" },
+  {
+    value: "defendantRespondentAttorney",
+    label: "Defendant / Respondent's Attorney"
   }
+];
+
+export default function MotionToWaiveFees_Web({ data }) {
+  return (
+    <Scoped css={css}>
+      <FormThatPrints>
+        <h1 className="web-from-input">This is a private record</h1>
+        <Section name="1. Personal information">
+          <TextInput dataKey="person.firstName" label="First name" />
+          <TextInput dataKey="person.middleName" label="Middle name" />
+          <TextInput dataKey="person.lastName" label="Last name" />
+          <TextInput dataKey="person.address" label="Street address" />
+          <TextInput dataKey="person.city" label="City" />
+          <TextInput dataKey="person.state" label="State" />
+          <TextInput dataKey="person.zip" label="Zip" />
+          <TextInput dataKey="person.phone" label="Phone" />
+          <TextInput dataKey="person.email" label="Email" />
+          <small className="web-form-input">
+            Check your email. You will receive information and documents at this
+            email address.
+          </small>
+
+          <Select
+            dataKey="person.relation"
+            label="Relation"
+            options={waiveFeeRelationship}
+          />
+
+          {data.person.relation &&
+            data.person.relation.includes("Attorney") && (
+              <TextInput label="Bar Number" dataKey="person.attorney" />
```

Comment:
Is this the first time that we are using `person.attorney` as a dataKey? If so, I'd prefer changing it to `person.attorney.barNumber`, so that if we need to capture the attorney's name or other info we have a whole object for the attorney instead of just a string barNumber.

Reply:
That makes sense. Will be more expressive when it comes to the dataKeys.